charts/spire/values.yaml

```diff
@@ -27,6 +27,9 @@ global:
     ## @param global.spire.strictMode Check values, such as trustDomain, are overridden with a suitable value for production.
     strictMode: false
 
+    ## @param global.spire.ingressControllerType Specify what type of ingress controller you're using to add the necessary annotations accordingly. If auto or blank, other is currently assumed. If other, no annotations will be added. Must be one of [ingress-nginx, other, auto] to override all settings.
+    ingressControllerType: ""
+
```

Comment:
Do we need the empty string here?

Reply:
Yes, or we could comment out the ingressControllerType all together.

It allows you to fall through to the individual ingresses settings, which default to auto. This can be useful when you have different ingress controller types within your k8s cluster, and need different ingress objects to target one or the other ingress controller. I have such a cluster.

Reply:
I mean why cant it be set to 'auto'? Seems like they are aliases.

Reply:
Oh ok, i get it now.

Reply:
if set, its not possible to unset or set it individually on each ingress. By it being blank/undefined, it passes  through to whatever is set at the individual level. it lets you do:

global.spire.ingressControllerType=ingress-nginx
or
spire-server.ingress.controllerType=traefik
spiffe-oidc-server.ingress.controllerType=ingress-nginx